Tests iOS/DerivablesUnitTest.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  DerivablesUnitTest.swift
+//  Tests iOS
+//
+//  Created by Angelos Veglektsis on 7/30/22.
+//
```

Comment:
Please note which tests it matches in the reference implementation and link back as well in a comment in the reference implementation.

Reply:
Fixed